app/assets/stylesheets/content/_tables.sass

```diff
@@ -377,3 +369,39 @@ table#time-report
       cursor: context-menu
     &.updated
       width: 80px
+
+.list--row
+  line-height: $base-line-height
+
+  &:nth-child(odd)
+    background-color: white
+
+    &:hover
+      background-color: #FFD
+
+  &:nth-child(even)
+    background-color: #f6f7f8
+
+    &:hover
+      background-color: #FFD
+
+.list--actions-cell
+  font-size: 0.85em
+  width: 15%
+  white-space: nowrap
+  text-align: right
+
+  & > .icon
+    padding: 0.2em 0.6em 0.2em 0
+    border: 1px solid #e4e4e4
+    border-radius: $button-radius
+    background-color: rgba(255, 255, 255, 0.8)
+    color: darken($content-link-color, 10%)
+
+    &:hover
+      border-color: $button-bg-hover-color
+      background-color: $button-bg-hover-color
+      color: white
```

Comment:
It looks wrong that the bg color is in a variable and the fg color is not...

What if I want a white bg on hover?

If you say it is too much effort to do it *right*, I'll buy it... but at least I need to ask ^^